packages/account/src/Components/poi/poi-upload-failed/upload-failed.jsx

```diff
@@ -4,7 +4,7 @@ import { Icon } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import IconMessageContent from 'Components/icon-message-content';
 
-const UnsupportedFailed = ({ error }) => (
```

Comment:
Is this change needed? this component naming should be based on the status of POI upload.

Reply:
yes, I agree, that’s why I decided to change this name.this component is used inside DetailComponent.  IMHO this name is more explicit, as inside this component we see the message 'Proof of identity documents upload failed' and icon 'IcPoiFailed'.  And we have several other components with similar purpose and behaviour (expired, limited, not-required etc.)